### DISCUSSION TITLE
Sub Table to Sub Table Task

### DISCUSSION THREAD

**Discussion by ghfsystem** [AntlerVC/firetable#822] [Support / Q&A] [answered]
How should i reference in a Task that I am trying to create to achieve creating a new record in another Sub Table?

I have Users/Notifications as a Sub collection in my firebase, so I added a Column of type Sub Table in my Users Table.

Similarly, I have Social Feed and a Sub Collection named Comments under it, where I record the comments that a post has received.

I am trying to configure a Task -> on Create on Sub Table COmments, Create a record in Notifications. 

Can you suggest the approach I should take?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @ghfsystem, sorry I missed this earlier. You can use ref as follows:
```
ref.collection(sub collectionName).add({})
```
For more use cases like these, check out the demo apps built using Rowy here: https://demo.rowy.io/
